plugins/mcreator-core/datalists/eventparameters.yaml

```diff
@@ -0,0 +1,4 @@
+- DAMAGE_AMOUNT:
+  type: number

```

Comment:
I would consider adding readable_name's to those that clarify to which trigger they belong, otherwise user will have to try until they find the correct one

Reply:
The readable_name 's format need to be thought.
eg.
- [TriggerName]-[ParameterName]

Reply:
One idea could be:

> Readable trigger name - PARAMETER_NAME

Reply:
> Readable_name can not use localization :<

Not a topic of this PR

> modify readable_name's code is not too relate for the PR

It is very related to this PR, as otherwise the PR will be hard to use which we can't accept in the core

Reply:
> > Readable_name can not use localization :<
> 
> Not a topic of this PR

I think it is in the topic. because trigger's readable_name will change with the UI's language.

Reply:
This PR should implement hardcoded names in readable_name for English

> I think it is in the topic

I appreciate your opinion, but this PR adds "Event parameter modifier to make some advanced operation" and not "Support for readable name" translations. While this may be something added in another PR, this is not on topic of this PR